addons/orioledb/templates/clusterdefinition.yaml

```diff
@@ -33,16 +33,6 @@ spec:
               - -c
             args:
               - curl -s http://$(KB_REPLICATION_PRIMARY_POD_FQDN):8008/switchover -XPOST -d '{"leader":"$(KB_REPLICATION_PRIMARY_POD_NAME)"}'
-      customLabelSpecs:
-        - key: apps.kubeblocks.postgres.patroni/scope
-          value: "$(KB_CLUSTER_NAME)-$(KB_COMP_NAME)-patroni$(KB_CLUSTER_UID_POSTFIX_8)"
-          resources:
-            - gvk: "v1/Pod"
-              selector:
-                app.kubernetes.io/managed-by: kubeblocks
-            - gvk: "apps/v1/StatefulSet"
-              selector:
-                app.kubernetes.io/managed-by: kubeblocks
```

Comment:
need add
```
  labels:
    apps.kubeblocks.postgres.patroni/scope: "$(KB_CLUSTER_NAME)-$(KB_COMP_NAME)-patroni$(KB_CLUSTER_UID_POSTFIX_8)"
```

on cmpd ? see pg's cmpd

Reply:
and env

Reply:
OK, i will submit a PR to add the cmpd for orioledb later.